### PR TITLE
adds combine

### PR DESCRIPTION
**Pull request by stoeffel**
how do you like this guys?

//cc @tomekwi @hemanth

### CODE REVIEW

File: tests/combine.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+test('#combine', () => {
+  const camelize = curry(combine)((a, b) => a + b.toUpperCase());
+  const dasherize = curry(combine)((a, b) => `${a}-${b.toLowerCase()}`);
+
+  equal(camelize('hello-world'), 'helloWorld');
+  equal(camelize('hello_world'), 'helloWorld');
+  equal(camelize('hello - world'), 'helloWorld');
+  equal(camelize('hello.world'), 'helloWorld');
+	equal(camelize('hello + world'), 'hello + world');
```
Comment (hemanth) [inline tests/combine.js]: Indent.

**Comment by tomekwi** [comment]
Hmm, I have mixed feelings. I don’t know if it’s not too specific for such a general-purpose library.

And apart from that – understanding the source took me a good minute or two – while with the other functions I grip it in seconds.

**Comment by stoeffel** [comment]
Hmm... maybe you are right. I thought it is general enough ( I wouldn't add camelize and stuff)... I will think about moving it to its own module.

**Comment by tomekwi** [comment]
:+1: it’s definitely a useful thing :)

**Comment by stoeffel** [comment]
I will keep this open until I moved it to its own module

**Comment by hemanth** [comment]
@stoeffel  move it to fp-js ;)

**Comment by stoeffel** [comment]
Closed in favor of https://github.com/stoeffel/reassemble-string

**Comment by stoeffel** [comment]
@hemanth I think this is best fitted as its own module

**Comment by hemanth** [comment]
Yup, all I was trying to do there was bring up back to `fp-js` ;)

**Pull request closed without merging**